book/modules/ROOT/pages/introduction.adoc

```diff
@@ -7,27 +7,39 @@
 
-This book is broken down into "core" topics. These topics are listed in the options below. They range from getting started with data science to information for the corporate partners who want to sponsor a project with us. 
-
-In addition, the live version of the "core" book contains the most up-to-date slate of projects students in The Data Mine are working on. All projects from previous semesters live in the xref:projects:ROOT:introduction.adoc[Projects Archive] appendix and are open to the public.
-
 [TIP]
 ====
-If you're here to explore or learn more about data science we recommend starting with the `Data Science` button below. Have fun!
+Where to go for specific topics:
+
+* Question about predictive models, data analysis, or visualization? Check out `Data Science`.
+* Need to know more about containers or research computing? Check out `Data Engineering`. 
+* Learning R or Python. `Coding Languages` might be helpful. 
+* Want more info on The Data Mine, corporate partners projects, or student course resources? Check out `Corporate Partners`.
+* Looking to see the projects that we use to build skills in analytics? Read through the `Course Links`.
+* Have a different questions or want to add content to The Examples Book? `Helpful Links` is the place for you. 
 ====
 
 == Core Topics
 
 xref:data-science:intro-to-ds:introduction.adoc[[.custom_button]#Data Science#]
 xref:data-engineering:intro-to-data-engineering:introduction.adoc[[.custom_button]#Data Engineering#]
+xref:coding-languages:ROOT:introduction.adoc[[.custom_button]#Coding Languages#]
 xref:crp:ROOT:introduction.adoc[[.custom_button]#Corporate Partners#]
-xref:think-summer:ROOT:introduction.adoc[[.custom_button]#Think Summer#]
 
 == Course links
 
-The following links are useful for accessing the course materials like the syllabus, office hours schedule, videos, projects, and more.
-
-xref:book:projects:19000-s2022-projects.adoc[[.custom_button]#STAT 19000#]
-xref:book:projects:29000-s2022-projects.adoc[[.custom_button]#STAT 29000#]
-xref:book:projects:39000-s2022-projects.adoc[[.custom_button]#STAT 39000#]
+xref:projects:current-projects:19000-s2022-projects.adoc[[.custom_button]#STAT 19000#]
+xref:projects:current-projects:29000-s2022-projects.adoc[[.custom_button]#STAT 29000#]
+xref:projects:current-projects:39000-s2022-projects.adoc[[.custom_button]#STAT 39000#]
 xref:projects:ROOT:introduction.adoc[[.custom_button]#Projects Archive#]
+
+== Workshops and Trainings
+
+xref:workshops:gallygoogle:introduction.adoc[[.custom_button]#Gallaudet Workshop#]
+xref:think-summer:ROOT:introduction.adoc[[.custom_button]#Think Summer#]
+
+== Helpful Links
+
+xref:book:FAQs:faqs.adoc[[.custom_button]#FAQ#]
+xref:book:ROOT:how-to-contribute.adoc[[.custom_button]#How to Contribute#]
+xref:book:contributors:contributors.adoc[[.custom_button]#Contributors#]
```

Comment:
Could we add our super-awesome-amazing datamine video to the bottom of the page or maybe even the top? The one that got a lot of views on youtube? That would be amazing I think. Maybe at the top. https://docs.antora.org/antora/latest/asciidoc/embed-video/#add-a-youtube-or-vimeo-video